environments/flow/optional.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Js-coding-standards
+ *
+ * @author      Robert Rossmann <rr.rossmann@me.com>
+ * @copyright   2016 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  extends: './recommended.js',
+
+  rules: {
+    // Enforces a particular style for boolean type annotations.
+    'flowtype/boolean-style': ['warn', 'boolean'],
```

Comment:
I think this could be a recommended rule with these settings. Consistency is of great importance.